openfecwebapp/api_caller.py

```diff
@@ -45,6 +45,40 @@ def load_search_results(query, query_type='candidates'):
     return results['results'] if len(results) else []
 
 
+def load_legal_search_results(query, query_type='all', limit=20):
+    filters = {}
+
+    def _transform_advisory_opinion(advisory_opinion):
```

Comment:
These nested functions don't seem to rely on variables defined in the outer context, so do they need to be nested? Prefer shorter, flatter functions, all things being equal: "flat is better than nested".